service/history/config/config.go

```diff
@@ -459,6 +460,7 @@ func New(dc *dynamicconfig.Collection, numberOfShards int, storeType string, isA
 		ReplicatorProcessorMaxRedispatchQueueSize:             dc.GetIntProperty(dynamicconfig.ReplicatorProcessorMaxRedispatchQueueSize, 10000),
 		ReplicatorProcessorEnablePriorityTaskProcessor:        dc.GetBoolProperty(dynamicconfig.ReplicatorProcessorEnablePriorityTaskProcessor, false),
 		ReplicatorProcessorFetchTasksBatchSize:                dc.GetIntPropertyFilteredByShardID(dynamicconfig.ReplicatorTaskBatchSize, 25),
+		ReplicatorUpperLatency:                                dc.GetIntProperty(dynamicconfig.ReplicatorTaskBatchSize, 40),
```

Comment:
`dynamicconfig.ReplicatorUpperLatency`?

also, shall we use `dc.GetDurationProperty` instead?

Reply:
updated